big_tests/tests/sm_SUITE.erl

```diff
@@ -469,13 +467,11 @@ resend_unacked_after_resume_timeout(Config) ->
     %% kill alice connection
     escalus_connection:kill(Alice),
 
-    U = proplists:get_value(username, AliceSpec),
-    S = proplists:get_value(server, AliceSpec),
-    1 = length(rpc(mim(), ejabberd_sm, get_user_resources, [U, S])),
-    %% wait 2 times longer to be sure that c2s is dead
-    ct:sleep({seconds, 2 * ?SHORT_RESUME_TIMEOUT}),
+    1 = length(get_user_alive_resources(AliceSpec)),
+    %% wait a bit longer to be sure that c2s is dead
```

Comment:
Don't use a predefined sleep time in cases like this, when we wait for a certain event to happen, when it's possible to monitor / verify when the event happens. [In this case we can use a monitor, e.g. like here](https://github.com/esl/MongooseIM/blob/fe8004d20f6cddc673b5eae4a0b1ab3838aea735/big_tests/tests/sm_SUITE.erl#L368-L373) or using [`wait_until_disconnected`](https://github.com/esl/MongooseIM/blob/fe8004d20f6cddc673b5eae4a0b1ab3838aea735/big_tests/tests/sm_SUITE.erl#L993).

I know that you're just modifying existing code, but let's try to leave it in a better shape than it was before.